.eslintrc

```diff
@@ -1,3 +1,14 @@
 {
-  "extends": "appium"
+  "extends": "@appium/eslint-config-appium",
+  "overrides": [
+    {
+      "files": "packages/fake-driver/**/*",
+      "rules": { "require-await": "off" }
+    },
+    { "files": "packages/support/**/*", "globals": { "BigInt": "readonly" } },
+    {
+      "files": "packages/*/test/**/*",
+      "rules": { "func-names": "off" }
+    }
+  ]
```

Comment:
@jlipps 

this file consolidates all `.eslintrc` files across the packages.

however, the `lint` gulp task also does whatever it does.  I'm a little concerned `eslint --fix` won't have the same result as `gulp lint --fix`.  it's _probably_ not a problem, but we should keep an eye open for linting churn.